docs/plugins/tv.md

```diff
@@ -0,0 +1,140 @@
+# TV
+
+The TV plugin serves as an abstraction layer to the _live TV_ functionality of a STB. The interface gives you access to the current channel
+and program information, and allows you to change the TV channel from inside an App.
```

Comment:
Might need to nuance this or elaborate. Changing TV channels from within an app is a restricted operation. Not every, or very likely most of the applications, will not be able to change the current Live TV Channel. However if the application is whitelisted to use this the functionality is available.

Reply:
good point! will update

Reply:
@wouterlucas why would we do this? Every app should be able to switch channels, there is no restriction needed for this.